_studio/shared/asc/src/asc.cpp

```diff
@@ -1366,8 +1367,8 @@ mfxStatus ASC::RunFrame(SurfaceIndex *idxFrom, mfxU32 parity) {
     res = m_queue->DestroyEvent(e);
     SCD_CHECK_CM_ERR(res, MFX_ERR_DEVICE_FAILED);
 
-    AscFrameAnalysis();
 
+    AscFrameAnalysis();
```

Comment:
You don't modify anything in this file actually. Please, remove the file from the review.